addons/account/models/chart_template.py

```diff
@@ -186,6 +188,18 @@ def _load(self, template_code, company, install_demo):
         self._load_translations(companies=company)
         self._post_load_data(template_code, company, template_data)
 
+        # Add translations for names of new accounts without xmlid.
+        # This is (usually) i.e. necessary for the following accounts
+        #     * (some) bank utility accounts (see function _setup_utility_bank_accounts)
+        #     * cash / bank journal accounts (created "by" account.journal)
+        #     * unaffected earnings accounts (created "by" res.company)
+        ids_of_accounts_with_xmlid = self.env['ir.model.data'].search([('model', '=', 'account.account')]).mapped('res_id')
+        new_accounts_without_xmlid = self.env['account.account'].search([
+            ('company_id', '=', company.id),
+            ('id', 'not in', preexisting_accounts.ids + ids_of_accounts_with_xmlid),
+        ])
+        self._load_account_name_code_translations(new_accounts_without_xmlid)
```

Comment:
Don't we want to also translate those when we install a new language?
It would fit better in `<account.chart.template>._load_translations` IMO